src/main/java/ch/astorm/jotlmsg/io/FlatEntryStructureFactory.java

```diff
@@ -0,0 +1,23 @@
+package ch.astorm.jotlmsg.io;
+
+import java.lang.reflect.InvocationTargetException;
+import java.nio.ByteBuffer;
+
+/**
+ * Creates FlatEntryStructures.
+ * 
+ * @author Guido Stein
+ *
+ */
+public class FlatEntryStructureFactory<T extends FlatEntryStructure> {
```

Comment:
Is this class really relevant ? Why not simply do `new MyFlatEntryStructure(bf)` where needed ?

Reply:
The replyto data is stored within the msg file like an array of structs in C. So FlatEntryListStructure and FlatEntryStructures provides some basis functionality like memory alignment and access to the binary data. OneOffEntryIDStructure is a special kind of FlatEntryStructure to interpret the data we need. But FlatEntryListStructure and FlatEntryStructures shouldn't work only with OneOffEntryIDStructure. So FlatEntryStructureFactory contains some generics to instantiate subclasses of FlatEntryStructure.

Reply:
OK thanks for the details. Let's keep it !